### PR TITLE
test/keccak-normalize-table: unittest for lookup tables

### PR DESCRIPTION
**Pull request by naure**
This PR verifies the properties of the lookup tables used in Keccak. Also minor refactoring for testability.

### CODE REVIEW

**Comment by lispc** [comment]
the degree is the limit of gate poly degree, not related to the circuit height at all..

File: zkevm-circuits/src/keccak_circuit/util.rs [hunk 1]
```diff
@@ -462,3 +483,188 @@ pub(crate) fn extract_field<F: FieldExt>(value: Value<F>) -> F {
...
+    }
+
+    // ---- Helpers ----
+
+    fn build_table(circuit: &TableTestCircuit) -> Vec<(u64, u64)> {
+        let prover = MockProver::<F>::run(circuit.log_height as u32, circuit, vec![]).unwrap();
+
+        let columns = prover.fixed();
+        assert_eq!(columns.len(), 2);
+        let unused_rows = 6;
```
Comment (lispc) [inline zkevm-circuits/src/keccak_circuit/util.rs:586]: for this magic number, either add some comments, or use ConstrainSystem.mininum_rows() ?

**Review by lispc — APPROVED** [review]
LGTM. we can merge it after ci passes.

File: zkevm-circuits/src/keccak_circuit/util.rs (hunk 1 above)
Comment (naure) [reply zkevm-circuits/src/keccak_circuit/util.rs:586]: done

**Pull request merged**